platform/security/handler/security-handler-saml/src/test/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandlerTest.java

```diff
@@ -176,7 +177,7 @@ public void testGetNormalizedTokenFromSession() throws Exception {
         SecurityTokenHolder tokenHolder = mock(SecurityTokenHolder.class);
         when(session.getAttribute(SecurityConstants.SAML_ASSERTION)).thenReturn(tokenHolder);
         SecurityToken securityToken = mock(SecurityToken.class);
-        when(tokenHolder.getSecurityToken()).thenReturn(securityToken);
+        when(tokenHolder.getSecurityToken(anyString())).thenReturn(securityToken);
```

Comment:
Instead of this, mock `request.getAttribute(ContextPolicy.ACTIVE_REALM)` to return a specific realm string, like 'foo', and then change this from `anyString()` to that string.